### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading '1')

### ISSUE THREAD

**Issue by AGNQuasar** [EDCD/coriolis#714]
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
Path: /outfit/krait_mkii
Error: Unknown
Details:
scriptUrl: https://coriolis.io/main.js
line: 46
col: 84922
error: {}

**Comment by Roon750** [comment]
It occurs when clicking on weapon modifications. Thanks

**Comment by AGNQuasar** [comment]
Yes, I totally forgot to mention that. Also, it only occurs when creating a
new build. When I viewed the already created build, this problem was not
encountered. Well, until I cleared the cache. The problem still persists.
And another one is added to the mix - when clicking on "Materials Needed
For This Build".

Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36
(KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
Path: /outfit/krait_mkii
Error: Unknown
Details:

scriptUrl: https://coriolis.io/main.js
line: 65
col: 247497
error: {}
